docs/netavark-firewalld.7.md

```diff
@@ -0,0 +1,90 @@
+% NETAVARK-FIREWALLD 5 Netavark Firewalld Interactions Man Page
```

Comment:
The number here needs to be updated too.

Reply:
Damn it, fixed